### ISSUE TITLE
Constant urgency coefficient for distant tasks

### ISSUE THREAD

**Issue by renan-cunha** [GothenburgBitFactory/taskwarrior#2174]
Is there a way to make urgency coefficient of taskwarrior, more specifically those about due dates, more precise?

For tasks with 2 months away for instance, it just puts the urgency constant, no matter if a task is 2 months away or 1 year.

**Comment by pbeckingham** [comment]
No, that would require a code change.  It currently only considers the range from 7 days ago to three weeks from now.
